src/cli/peakdetector/peakdetectorcli.cpp

```diff
@@ -857,19 +857,18 @@ QString PeakDetectorCLI::_getRedirectionUrl(QString datetimestamp,
     QString redirectionUrl = "";
     switch (_currentPollyApp) {
     case PollyApp::PollyPhi: {
+        QString workflowId = 
+            _pollyIntegration->obtainComponentId(PollyApp::PollyPhi);
```

Comment:
We have a different function for obtaining workflow Id. So, this should be

```C++
_pollyIntegration->obtainWorkflowId(PollyApp::PollyPhi);
```

I am experiencing a crash due to this when I run CLI. The failure to obtain `workflowId` should still not crash, and can be handled better.